content/_partials/books.adoc

```diff
@@ -0,0 +1,3 @@
+NOTE: The list of books on this page is provided for informational purposes.
+The books are not endorsed by Jenkins project.
+Older books on the list contain some outdated information.
```

Comment:
```suggestion
[NOTE]
====
The books in this list are not endorsed by the Jenkins project.
The list of books on this page is provided for informational purposes, sorted from newest to oldest publication date.
Since older books on the list can contain outdated information, the link:/book/getting-started/[Jenkins documentation] can  help fill in any gaps.
====
```
I think the admonition block could be used in this case, since there are multiple sentences being added
I also tried to update the syntax so that it might read/explain a bit better, and rearranged the information so that the first line states that the books are not endorsed by the project. Also added a link to the jenkins documentation to help with older books/outdated information.